homeassistant/components/alarmdecoder.py

```diff
@@ -40,6 +46,11 @@
 DEFAULT_DEVICE_TYPE = 'socket'
 DEFAULT_DEVICE_HOST = 'localhost'
 DEFAULT_DEVICE_PORT = 10000
+DEFAULT_DEVICE_SSL = False
+DEFAULT_DEVICE_SELF_SIGNED = False
```

Comment:
It's not clear to me why all these options are needed. In other platforms we just have a `ssl_verify` option.

Reply:
The default behavior in alarmdecoder is to verify and I wanted to preserve that functionality in the default configuration. You can use SSL without it being a self signed certificate by just leaving the self signed option out entirely.